front/components/assistant/CreateAgentButton.tsx

```diff
@@ -49,9 +49,9 @@ export const CreateAgentButton = ({
           disabled={isLoading}
         />
       </DropdownMenuTrigger>
-      <DropdownMenuContent>
+      <DropdownMenuContent align="start">
         <DropdownMenuItem
-          label="agent from scratch"
+          label="Agent from scratch"
```

Comment:
I thiiink it looks more natural when A is capitalized?

Reply:
I agree with you but there's been a discussion around it and Gab wants to keep it.

Reply:
alright alright 🙃